src/commands/soundbot/AddCommand.ts

```diff
@@ -0,0 +1,66 @@
+import config from '../../../config/config.json';
+
+import fs from 'fs';
+import https from 'https';
+
+import { MessageAttachment } from 'discord.js';
+
+import BaseCommand from '../base/BaseCommand';
+
+import SoundUtil from '../../util/SoundUtil';
+
+export class AddCommand extends BaseCommand {
+  public run() {
+    let error: string | undefined;
```

Comment:
Does this compile without the type annotation? If not, can you try `let error?: string`?

Reply:
Oh lol it does. I'm pretty sure there was it time where it didn't...